docs/GLOSSARY.md

```diff
@@ -1,122 +1,180 @@
 # Glossary
 
-Index of terminology used in Randovania. Certain terms sometimes are used only in the GUI, with more technical ones being used in code or CLI.
+Index of terminology used in Randovania. Some terms are used only in the GUI, with more technical ones being used in code or CLI.
+
+## Action
+
+Anything the [Generator](#generator) or [Solver](#solver) can do which alters the game's state. Actions include obtaining [Resources](#resouce), crossing [Connections](#connection) between [Nodes](#node), and using [Docks](#dock) or [Teleporters](#teleporter).
```

Comment:
Actions are only a concept of the generator, and they're one of:
- Assigning Pickups
- Collecting a ResourceNode that isn't safe.

For obvious reasons, the second one is considered dangerous.

Reply:
Rewrote this